sdk/resourcemanager/Azure.ResourceManager.Core/samples/Sample1_HelloWorld.md

```diff
@@ -0,0 +1,29 @@
+# Example: Getting a susbcription
+
+>Note: Before getting started with the samples, make sure to go trought [prerequisites guide](<--./docs/Prerequistes.md-->).
+
+The following code shows how to get the default subscription:
+
+```csharp Snippet:HelloWorld_GetDefaultSubscription
+ArmClient armClient = new ArmClient(new DefaultAzureCredential());
+Subscription subscription = armClient.DefaultSubscription;
+Console.WriteLine(subscription.Id);
+```
+
+It is possible to get an specific subscription as it's shown next:
+
+``` csharp Snippet:HelloWorld_GetSpecificSubscription
+var subscriptionId = "db1ab6f0-4769-4b27-930e-01e2ef9c123c";
+ArmClient armClient = new ArmClient(new DefaultAzureCredential());
+Subscription subscription = armClient.GetSubscriptions().TryGet(subscriptionId);
```

Comment:
use async in examples

Reply:
Got it. 
According to [this guideline](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-sync-samples-naming), there should be an Async version of the Hello World.

I'm adding the Async variant for this method in a separate `Sample1_HelloWorldAsync.md`